scripts/utils.js

```diff
@@ -0,0 +1,105 @@
+const fsp = require("fs").promises;
```

Comment:
This file was created to bring this repo closer to the `remix` repo. All of these utilities are duplicated in both places, and in Remix we already keep them in a separate utils file. This will simplify merging the two.